### ISSUE TITLE
Missing mutation documentation

### ISSUE THREAD

**Issue by offjeff91** [Shopify/shopify-api-js#676]
Hi, I need to run a mutation into shopify but so far I only found in your [documentation](https://github.com/Shopify/shopify-api-js/blob/main/docs/reference/clients/Graphql.md) how to run queries. I didn't find any parameter about mutation into index.ts and neither any mention in your doc. Is that possible?

Thanks.

**Comment by mkevinosullivan** [comment]
@offjeff91 Here's an [example](https://github.com/Shopify/shopify-app-template-node/blob/main/web/product-creator.js#L73-L81) of a mutation used to populate products on a store from our default template.

In general, using this [example](https://github.com/Shopify/shopify-api-js/blob/main/docs/reference/clients/Graphql.md#using-a-query-string) from the docs, the following
```js
const response = await client.query({
  data: `{
    products (first: 10) {
      edges {
        node {
          id
          title
          descriptionHtml
        }
      }
    }
  }`,
});
```
can also be written
```js
const response = await client.query({
  data: `query GetFirstTenProducts {
    products (first: 10) {
      edges {
        node {
          id
          title
          descriptionHtml
        }
      }
    }
  }`,
});
```

In other words, the `query...` part can be omitted as it's the default expected in a GraphQL query.  However, a mutation requires the `mutation` keyword, e.g.,
```js
const response = await client.query({
  data: {
    query: `mutation populateProduct($input: ProductInput!) {
      productCreate(input: $input) {
        product {
          id
        }
      }
    }`,
    variables: {
      input: {
        title: 'My totally cool new product title',
        variants: [{ price: 1000.0 }],
      },
    },
  },
});
```

I hope this helps get you started on using mutations.

**Comment by offjeff91** [comment]
Thanks, @mkevinosullivan! I really appreciate.

**Comment by falcon0119** [comment]
client.query


I think the client only has request and fetch and not query.
